resources/validations/docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: "3.9"
+
+services:
+  validator:
+    build: .
+    volumes:
+      - .:/root
```

Comment:
NB: I did not know I could do `.:/root` notation in the docker-compose.yml file. This approach is useful. Thanks for educating me!